sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample2_RecognizeCustomFormsFromFile.cs

```diff
@@ -46,12 +46,12 @@ public async Task RecognizeCustomFormsFromFile()
                     {
                         Console.WriteLine($"Field '{field.Name}: ");
 
-                        if (field.LabelText != null)
+                        if (field.LabelData != null)
                         {
-                            Console.WriteLine($"    Label: '{field.LabelText.Text}");
+                            Console.WriteLine($"    Label-------: '{(string)field.LabelData}");
```

Comment:
Why adding the `-------`? You don't seem to be doing the same thing in `Sample2_RecognizeCustomFormsFromUri.cs`.

Reply:
Also, I don't think that using the implicit operator here is preferred since we're forcing a cast. It's convenient when we have:
```cs
string text = field.LabelData;
```
But here, calling `field.LabelData.Text` seems easier to understand.

Reply:
left by mistake, undo